care/facility/api/serializers/patient.py

```diff
@@ -2,12 +2,20 @@
 
 from django.db import transaction
 from django.utils.timezone import make_aware
+from phonenumber_field.serializerfields import PhoneNumberField
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'phonenumber_field.serializerfields'](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5332931)